ivadomed/utils.py

```diff
@@ -313,27 +325,30 @@ def run_eval(self):
         return dct
 
 
-def save_nii(data_lst, z_lst, fname_ref, fname_out, slice_axis, debug=False, unet_3D=False, binarize=True):
-    """Save the prediction as nii.
-        1. Reconstruct a 3D volume out of the slice-wise predictions.
-        2. Re-orient the 3D array accordingly to the ground-truth orientation.
+def pred_to_nii(data_lst, z_lst, fname_ref, fname_out, slice_axis, debug=False, kernel_dim='2d', bin_thr=0.5):
+    """Convert the NN predictions as nii.
```

Comment:
is there an explicit convention in ivadomed project as to what a "nii" object is? is it a nibabel object? or a nifti file name?

Reply:
Good point, I am going to rename this as `pred_to_nibabel` or `pred_to_nib`.